tasks/patternpack.js

```diff
@@ -134,6 +139,24 @@ module.exports = function (grunt) {
     }
   }
 
+  function ensureFilesExist(options) {
+    var path = require("path");
+
+    // Create core css file if it does not exist
+    var coreCssPath = options.assets + "/" + options.css.preprocessor + "/" + options.css.fileName + "." + options.css.preprocessor;
```

Comment:
Technically we pass `sass` in as the option for Sass, but the proper file extension would be `.scss` because `.sass` is the other bracketless syntax.

Maybe we should allow either `scss` or `sass` as appropriate values? Because `grunt-sass` technically just looks at the file extension to decide which to use, I believe.